reverse_proxy/templates/generic.conf.j2

```diff
@@ -5,7 +5,13 @@ server {
   # Internal only endpoint for performing a proxy pass to the cran mirror.
   location /{{ generic_name }}-fetch {
     internal;
-    proxy_pass {{ upstream_endpoint }};
+    rewrite_by_lua_block {
+      local uri = ngx.var.uri:gsub("{{ generic_name }}%-fetch", "")
+      local curl = io.popen("/usr/bin/curl \"{{ upstream_endpoint }}" .. uri .. "\"")
```

Comment:
`ngx.var.uri` is the nornalised unescaped url so is vulnerable to exploits, but the current regexes should prevent this
https://github.com/hic-infra/shared-services-ansible-roles/blob/08bf395c88b9430b085b7e974cb0690065d206c7/reverse_proxy/defaults/main.yml#L7-L11